README.md

```diff
@@ -234,6 +237,7 @@ For examples:
 | `examples/data-sources/<data source name>/data-source<*>.tf`                 | Data source example config(s)              |
 | `examples/ephemeral-resources/<ephemeral resource>/ephemeral-resource<*>.tf` | Ephemeral resource example config(s)       |
 | `examples/functions/<function name>/function<*>.tf`                          | Function example config(s)                 |
+| `examples/list-resources/<list resource>/list-resource.tf`                   | List resource example config               |
```

Comment:
The code looks like this should be `.tfquery.hcl`, as well as we should add the optional wildcard `<*>` here like the other resource examples 👍🏻